subtitle-downloader/app.py

```
@@ -0,0 +1,149 @@
+    """Handle 500 errors"""
+    logger.error(f"Server error: {str(error)}")
+    return jsonify({'error': 'Internal server error'}), 500
+
+
+if __name__ == '__main__':
+    # Create downloads folder if it doesn't exist
+    os.makedirs(app.config['DOWNLOAD_FOLDER'], exist_ok=True)
+
+    # Run the Flask app
+    app.run(
+        host='0.0.0.0',
```

Ruff (0.15.5)

[error] Possible binding to all interfaces

(S104)